recipes/easysnowdata/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "easysnowdata" %}
+{% set version = "0.0.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/easysnowdata-{{ version }}.tar.gz
+  sha256: e392d14b66d0827f4ea4cbc64dc0257d45908d8c6bd77d4466e0a0ccbc1a15e3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=64
+    - setuptools-scm >=8
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - matplotlib-base
+    - pandas
+    - geopandas
+    - rasterio
+    - xarray
+    - rioxarray
+    - shapely
+    - pystac-client
+    - planetary-computer
+    - odc-stac
+    - xee
+    - beautifulsoup4
+    - py3dep
+    - earthaccess
+    - contextily
+    - folium
+    - mapclassify
+    - lxml
+    - pytkdocs
+    - scikit-image
+
+test:
+  imports:
+    - easysnowdata
+  commands:
+    - pip check
+  requires:
+    - pip
```

```suggestion
    - python {{ python_min }}
    - pip
```

One last item and then I'm happy to merge

---

fixed, thank you!!